src/new-user-experience/modules/configure-sync/index.js

```diff
@@ -347,6 +347,10 @@ export const ConfigureSync = ( {
 									) }
 									indent={ indent }
 									className="import-products-wrapper"
+									description={ __(
+										'Use this to bring new products from Square into WooCommerce. This is different from "Sync Now" which only updates products that are already synced.',
+										'woocommerce-square'
```

Comment:
"This is different from "Sync Now" which only updates products that are already synced." This is correct only when SOR is Square.